src/App.jsx

```diff
@@ -1,10 +1,36 @@
-import "./App.css";
+//import "./App.css";
+import { RouterProvider, createBrowserRouter } from "react-router-dom";
+import Navigation from "./components/Navigation";
+import PersonalPage from "./components/PersonalPage/PersonalPage";
+import Home from "./components/Home/Home";
 
 function App() {
   
+  const appRoutes = createBrowserRouter([
+    {
+      path: "/",
+      element: <Home/>
+    },
+    {
+      path: "/home",
+      element: <Home/>
```

Comment:
These paths here for "/" and "/home" are unecessary. I see what you were trying to do, but theres an easier way